core/templates/services/bottom-navbar-status.service.ts

```diff
@@ -38,7 +38,7 @@ export class BottomNavbarStatusService {
   isBottomNavbarEnabled(): boolean {
     return (
       this.bottomNavbarIsEnabled &&
-          this.windowDimensionsService.isWindowNarrow());
+          this.windowDimensionsService.getWidth() < 1000);
```

Comment:
Why this change? Should we change the method `isWindowNarrow` if it's not working correctly?

Reply:
No the ```isWindowNarrow``` is working perfectly fine, but the top navbar save icons kind of disappear at this range(1000px), and i wanted to make the bottom navbar appear as soon as the top navbar icons disappear and the isWindowNarrow works with ```768px```.